e2e/compose-env.yaml

```diff
@@ -7,7 +7,7 @@ services:
   engine:
       image: 'docker:${TEST_ENGINE_VERSION:-edge-dind}'
       privileged: true
-      command: ['--insecure-registry=registry:5000']
+      command: ['--insecure-registry=registry:5000', '--experimental']
```

Comment:
Doesn't this change the whole test suite?

Reply:
@cpuguy83 yes it does. I think it's fine right now, but we way want to run e2e test against experimental and non-experimental job, but then the changes would be heavier.

Reply:
The test would also need to ensure that experimental is enabled.
I think it may be prudent to support both modes in e2e and then skip tests that require experimental on non-experimental daemons rather than changing the suite for this test.